From 9a1db954cf12d180865dd6bc2e240561ecce13e2 Mon Sep 17 00:00:00 2001
From: Antonin Godard <antonin.godard@bootlin.com>
Date: Fri, 25 Oct 2024 17:19:00 +0200
Subject: [PATCH 1/3] Don't use a custom deployment directory

Using custom deployment directory is not following the standards.
Revert this change by meta-ti to give the good example to trainees.

Keep the definition of TI_COMMON_DEPLOY as it may be used by other
recipes.

Also, remove the change to TMPDIR done in the machine configuration,
to keep the temporary directory unified and avoid an error caused by
the change above.

Signed-off-by: Antonin Godard <antonin.godard@bootlin.com>
---
 meta-ti-bsp/conf/layer.conf | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/meta-ti-bsp/conf/layer.conf b/meta-ti-bsp/conf/layer.conf
index c8e4cd395992..5a425138867b 100644
--- a/meta-ti-bsp/conf/layer.conf
+++ b/meta-ti-bsp/conf/layer.conf
@@ -35,10 +35,10 @@ SIGGEN_EXCLUDERECIPES_ABISAFE += " \
 
 HOSTTOOLS_NONFATAL += "truncate xxd comm"
 
-TI_COMMON_DEPLOY ?= "${TOPDIR}/deploy-ti"
-DEPLOY_DIR_IMAGE:ti-soc = "${TI_COMMON_DEPLOY}/images/${MACHINE}"
-SDK_DEPLOY:ti-soc = "${TI_COMMON_DEPLOY}/sdk"
-PSEUDO_IGNORE_PATHS:append:ti-soc = ",${TI_COMMON_DEPLOY}"
+TI_COMMON_DEPLOY ?= "${TOPDIR}/tmp/deploy"
+# DEPLOY_DIR_IMAGE:ti-soc = "${TI_COMMON_DEPLOY}/images/${MACHINE}"
+# SDK_DEPLOY:ti-soc = "${TI_COMMON_DEPLOY}/sdk"
+# PSEUDO_IGNORE_PATHS:append:ti-soc = ",${TI_COMMON_DEPLOY}"
 
 MIRRORS += " \
     git://git.ti.com/.* git://github.com/TexasInstruments/BASENAME;protocol=https \
-- 
2.43.0

